validator/validator.go

```diff
@@ -19,10 +19,10 @@ type Validator struct {
 	allowedClockSkew   time.Duration                              // Optional.
 }
 
-// UserContext is the struct that will be inserted into
+// ValidatedClaims is the struct that will be inserted into
 // the context for the user. CustomClaims will be nil
 // unless WithCustomClaims is passed to New.
-type UserContext struct {
+type ValidatedClaims struct {
 	CustomClaims     CustomClaims
 	RegisteredClaims jwt.Claims
```

Comment:
Since this is now a generic validator, we should avoid using `jwt.Claims` in a public struct.

Reply:
Fixed in https://github.com/auth0/go-jwt-middleware/pull/121/commits/e0cfb1f971d7b143f6a20d415a5554e91979068a